src/shared/cloudformation/cloudformation.ts

```diff
@@ -9,6 +9,7 @@ import * as yaml from 'js-yaml'
 import * as filesystemUtilities from '../filesystemUtilities'
 import { SystemUtilities } from '../systemUtilities'
 import { getLogger } from '../logger'
+import { APIGatewayProperties } from '../sam/debugger/awsSamDebugConfiguration'
```

Comment:
is there precedent for taking on a dependency to `sam/debugger/*` from `cloudformation.ts`? If not, we should probably avoid it, so that `cloudformation.ts` remains relatively "pure". You can probably just copy-paste whatever `APIGatewayProperties['payload']` is pulling in. Same reason we don't reference `APIGatewayProperties['httpMethod']` even though it causes duplication.

Reply:
This is great info, thank you.  Updated.